### PR TITLE
libevent: 2.0.5

### PR DESCRIPTION
**Pull request by Chilledheart**

### CODE REVIEW

File: mingw-w64-libevent/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Maintainer: Alexey Pavlov <alexpux@gmail.com>
+
+_realname=libevent
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=2.0.22
+pkgrel=0
```
Comment (Alexpux) [inline mingw-w64-libevent/PKGBUILD]: pkgrel=0

File: mingw-w64-libevent/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+pkgdesc="An event notification library"
+arch=('i686' 'x86_64')
```
Comment (Alexpux) [inline mingw-w64-libevent/PKGBUILD]: arch=('any')

File: mingw-w64-libevent/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    --prefix=${MINGW_PREFIX} \
+    --build=${MINGW_CHOST} \
+    --host=${MINGW_CHOST} \
+    --target=${MINGW_CHOST} \
+    --sysconfdir=${MINGW_PREFIX}/etc
+  make
+}
+
+package() {
+  optdepends=('python2: to use event_rpcgen.py')
```
Comment (Alexpux) [inline mingw-w64-libevent/PKGBUILD]: ${MINGW_PACKAGE_PREFIX}-python2

**Comment by Chilledheart** [comment]
force-updated

**Pull request merged**